tau_embedding_settings.py

```diff
@@ -181,4 +194,823 @@ def setup_embedding(configuration: Configuration, scopes: List[str]):
+                            "p1_ptcut": 35,
+                            "p2_ptcut": 35,
+                            "p1_etacut": 2.1,
+                            "p2_etacut": 2.1,
+                            "p1_filterbit": 20,
+                            "p1_trigger_particle_id": 15,
+                            "p2_filterbit": 20,
+                            "p2_trigger_particle_id": 15,
+                            "max_deltaR_triggermatch": 0.4,
+                        },
+                        {
+                            "flagname": "trg_double_tau40_tightiso",
+                            "p1_ptcut": 40,
+                            "p2_ptcut": 40,
+                            "p1_etacut": 2.1,
+                            "p2_etacut": 2.1,
+                            "p1_filterbit": 20,
+                            "p1_trigger_particle_id": 15,
+                            "p2_filterbit": 20,
+                            "p2_trigger_particle_id": 15,
+                            "max_deltaR_triggermatch": 0.4,
+                        },
+                        {
+                            "flagname": "trg_double_tau40_mediumiso_tightid",
+                            "p1_ptcut": 40,
+                            "p2_ptcut": 40,
+                            "p1_etacut": 2.1,
+                            "p2_etacut": 2.1,
+                            "p1_filterbit": 20,
+                            "p1_trigger_particle_id": 15,
+                            "p2_filterbit": 20,
+                            "p2_trigger_particle_id": 15,
+                            "max_deltaR_triggermatch": 0.4,
+                        },
+                        {
+                            "flagname": "trg_double_tau35_tightiso_tightid",
+                            "p1_ptcut": 35,
+                            "p2_ptcut": 35,
+                            "p1_etacut": 2.1,
+                            "p2_etacut": 2.1,
+                            "p1_filterbit": 20,
+                            "p1_trigger_particle_id": 15,
+                            "p2_filterbit": 20,
+                            "p2_trigger_particle_id": 15,
+                            "max_deltaR_triggermatch": 0.4,
+                        },
+                    ],
+                    "2017": [
+                        {
+                            "flagname": "trg_double_tau40_tightiso",
+                            "hlt_path": "HLT_DoubleTightChargedIsoPFTau40_Trk1_eta2p1_Reg",
+                            "p1_ptcut": 40,
+                            "p2_ptcut": 40,
+                            "p1_etacut": 2.1,
+                            "p2_etacut": 2.1,
+                            "p1_filterbit": 6,
+                            "p1_trigger_particle_id": 15,
+                            "p2_filterbit": 6,
+                            "p2_trigger_particle_id": 15,
+                            "max_deltaR_triggermatch": 0.4,
+                        },
```

Comment:
The filterbits are only changed for 2018. Is this intended? (sane in mt)

Reply:
I focused on 22018 only for now, should be checked in 2017 and 2016